ksql-parser/src/main/java/io/confluent/ksql/parser/rewrite/StatementRewriter.java

```diff
@@ -395,21 +393,16 @@ protected Node visitCreateStream(final CreateStream node, final Object context)
   }
 
   protected Node visitCreateStreamAsSelect(final CreateStreamAsSelect node, final Object context) {
+    final Optional<Expression> partitionBy = node.getPartitionByColumn()
+        .map(exp -> (Expression) process(exp,context));
+
     return new CreateStreamAsSelect(
         node.getLocation(),
         node.getName(),
         (Query) process(node.getQuery(), context),
         node.isNotExists(),
-        node.getProperties().entrySet().stream()
-            .collect(Collectors.toMap(
-                Map.Entry::getKey,
-                e -> (Literal) process(e.getValue(), context)
-            )),
-        node.getPartitionByColumn().isPresent()
-            ? Optional.ofNullable(
-            (Expression) process(node.getPartitionByColumn().get(),
-                context))
-            : Optional.empty()
+        node.getProperties(),
```

Comment:
This does remove rewriting of the properties. However, we don't use this anyway and the same is already true for C* properties.